data-prepper-plugins/opensearch-source/src/main/java/org/opensearch/dataprepper/plugins/source/opensearch/configuration/IndexParametersConfiguration.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.source.opensearch.configuration;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+
+public class IndexParametersConfiguration {
```

Comment:
@ashoktelukuntla, should we be supporting document info as part of our configuration? We need to provide the index, document id, type in our events. Were we planning on including this by default or requiring the user to enable these feature? If we want to include this we may want to add this to the metadata attributes so the data is not polluted with this information.

Reply:
Doc info will be added in the next PR